flow2ml/Filters.py

```diff
@@ -131,6 +131,29 @@ def applygaussian(self,classPath):
         
         # saving the image.
         plt.imsave(classPath+"/GaussianImages/gaussian"+image, cv2.cvtColor(gaussian, cv2.COLOR_RGB2BGR))
+        
+        
+        def applybilateral(self,classPath):
+    ''' 
+      Applies Bilateral Filter to all the images in the given folder. 
+      Args : 
+        classPath : (string) directory containing images for a particular class.
+    '''
+    try:
+      os.mkdir(classPath+"/Bilaterally_FilteredImages")    
+    except:
+      pass
```

Comment:
Add proper exception handling message here

Reply:
could you help me a little on this one,what exacly do i need to add?

Reply:
Keep like this:
except Exception as e:
            print(f"Cannot create directory with given name due to {e}")